titus-server-master/src/test/java/com/netflix/titus/master/integration/v3/job/basic/JobSubmitAndControlBasicTest.java

```diff
@@ -84,8 +84,8 @@ public void testSubmitSimpleBatchJobWhichEndsOk() {
         JobDescriptor<BatchJobExt> tmpJob =
                 appendJobDescriptorAttribute(ONE_TASK_BATCH_JOB, JobAttributes.JOB_ATTRIBUTES_CREATED_BY, "embeddedFederationClient");
         tmpJob = appendJobDescriptorAttribute(tmpJob, JobAttributes.JOB_ATTRIBUTE_ROUTING_CELL, "embeddedCell");
+        tmpJob = appendJobDescriptorAttribute(tmpJob, JobAttributes.JOB_ATTRIBUTES_CALL_REASON,"test call");
```

Comment:
I think you should test that this value is added by the `BasicServiceJobAction`, not setting it in test here.
Please, find where the `updateJobCapacity` is tested, and verify that this attribute is actually set.